src/formik.tsx

```diff
@@ -91,14 +95,20 @@ export interface FormikState<V> {
 export interface FormikActions<P, V> {
   /* Manually set top level error */
   setError: (e: any) => void;
-  /* Manually set Errors */
+  /* Manually set errors object */
   setErrors: (errors: FormikErrors) => void;
   /* Manually set isSubmitting */
   setSubmitting: (isSubmitting: boolean) => void;
-  /* Manually set touched fields */
+  /* Manually set touched object */
   setTouched: (touched: FormikTouched) => void;
-  /* Manually set values  */
+  /* Manually set values object  */
   setValues: (values: V) => void;
+  /* Set value of form field directly */
+  setFieldValue: (field: string, value: any) => void;
```

Comment:
Hrmmm... React should batch the two calls together (in theory). Remember `setState` is just a _request_ to change state. https://facebook.github.io/react/docs/react-component.html#setstate

My thinking is that you are already into custom component land here, explicit > implicit. Maybe we  have this wrong and it should just be
```tsx
setField(field: string, partialStates: {
   value: any;
   touched: boolean;
   error: string
})
```
or we can throw our hands up and just give `setFormikState()` and let people do whatever.